.github/workflows/TestOnPRs.yml

```diff
@@ -0,0 +1,26 @@
+name: Test Pull Request
```

Comment:
I like it, separate workflow, reusing a resuable workflow - exactly how it's meant to be!

Reply:
👍 indeed, great suggestion.